pkg/cryptoinfo/info.go

```diff
@@ -13,18 +13,52 @@ type KeyInfo struct {
 
 // Maybe make these types?
 const (
-	kiPEM         = "PEM"
-	kiDER         = "DER"
-	kiCertificate = "CERTIFICATE"
+	kiPEM = "PEM"
```

Comment:
In Rust, it's common for this to be an `enum`, and then use that enum to pattern match against. Is that essentially what you're using this for?